internal/command/instance_model.go

```diff
@@ -1,15 +1,20 @@
 package command
 
 import (
+	"golang.org/x/text/language"
+
 	"github.com/caos/zitadel/internal/domain"
 	"github.com/caos/zitadel/internal/eventstore"
 	"github.com/caos/zitadel/internal/repository/instance"
-	"golang.org/x/text/language"
 )
 
 type InstanceWriteModel struct {
 	eventstore.WriteModel
 
+	Name            string
+	State           domain.InstanceState
+	GeneratedDomain string
```

Comment:
just a little thing, but do we really need the generated domain on the write model? what do we ensure with that
same for existing projectID and globalOrgID

Reply:
Not sure, we will have a look at this with @adlerhurst branch